docs/superpowers/plans/2026-05-04-jj-purge-git-workflow.md

```
@@ -0,0 +1,444 @@
+- [ ] **Step 2: Replace DCO sign-off instructions**
+
+Change:
+```markdown
+Add it with `jj describe`, including the sign-off line in the message:
+
+```
+jj describe -m "$(cat <<'EOF'
+Signed-off-by: Your Name <your.email@example.com>
+EOF
+)"
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)